src/providers/realtime-provider.js

```diff
@@ -75,6 +76,27 @@ export default class RealtimeProvider {
         };
     }
 
+    subscribeToLimits(domainObject, callback) {
+        const subscriptionDetails = this.buildSubscriptionDetails(domainObject, callback);
+        const id = subscriptionDetails.subscriptionId;
+        this.subscriptionsById[id] = subscriptionDetails;
+
+        if (this.connected) {
```

Comment:
So, we do this in the realtime provider as well, where we check `this.connected` and only send or queue if we're connected. In `sendOrQueueMessage` https://github.com/akhenry/openmct-yamcs/blob/master/src/providers/realtime-provider.js#L151 we already check if connected and queue it if not. I think we need to take this out of an if and just sendOrQueueMessage either way? We may need to update that in the realtime provider subscribe as well. I _think_ the affect of this would be if you try to subscribe to realtime (anything) and it's not connected, when it is it won't subscribe unless you refresh the view.

Reply:
Agreed. I'll remove it.